src/main/java/model/OperationType.java

```diff
@@ -0,0 +1,27 @@
+package model;
+
+public enum OperationType {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
Consider marking the 'code' field as `final` since it is intended to be immutable.